lisa/features/network_interface.py

```diff
@@ -38,7 +38,9 @@ def remove_extra_nics(self) -> None:
     def reload_module(self) -> None:
         raise NotImplementedError
 
-    def get_nic_count(self, is_sriov_enabled: bool = True) -> int:
+    def get_nic_count(
+        self, is_sriov_enabled: bool = True, include_primary: bool = True
```

Comment:
Can it just be got by minus 1?

Reply:
we are not sure, the primary one is sriov or synthetic?

Reply:
As the logic, it always finds a primary. What's different if it's sriov or synthetic? In this place, it looks like just need a number.

Reply:
If primary is sriov, others are synthetic, this method will return all sriov nic count, should we minus 1 or not? 
The code will get extra sriov nics and extra synthetic nics, then restore them after testing (which need update nics.)

Reply:
update